### ISSUE TITLE
Domain routing

### ISSUE THREAD

**Issue by inFureal** [skipperbent/simple-php-router#468]
When I code like this:
```php
Router::group(['domain' => "admin." . $domain], function () {

    Router::get("/", function () {

        return "Admin panel";

    })->name('admin.index');

});
```
it throws `NotFoundHttpException` "Route not found: "/"

But... When I code with 'variables' with regular expressions it's runnig perfectly.
```php
Router::group(['domain' => "{sub}." . $domain], function () {

    Router::get("/", function () {

        return "Admin panel";

    })->name('admin.index');

})->where(["sub" => "admin"]);
```
__Is it a bug?__

P.S. Sorry for my englandsky :)

**Comment by inFureal** [comment]
And when I run 
```php
SimpleRouter::getUrl('admin.index');
```
It returns `Route not found: "/{sub}.<domain>/"`. <domain> -it's my real domain, which I hide :)

**Comment by skipperbent** [comment]
Please post the simple-router debug information.

If you are using subdomains with special characters you might have to set your own regex to catch it.

**Comment by inFureal** [comment]
Sorry for late response.
I not logner maintaining that project. So this issue no longer actual. you can rely only on the available data.

**Comment by skipperbent** [comment]
Thanks for the reply! I will add some unit tests to see if I can replicate the issue. Can you remember the domain name you used - if it had special characters, was a local domain etc?

Thanks

**Comment by skipperbent** [comment]
Thanks for taking the time to report the issue. I found the bug and a fix will be available in the upcoming version 4.3.

**Comment by skipperbent** [comment]
This issue has been resolved in version 4.3.0.0.
Feel free to open another issue if you are still experiencing issues.
